docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>It looks like this title is level 1 markdown title which feels a bit weird to me and makes the toc on the right sidebar look a bit strange. I think this should be at the same level as the "introduction to aesara" section. Also, this image isn't rendering correctly either.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>+1</p>

Reply:
+1